sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Utils.java

```diff
@@ -70,8 +77,14 @@ static String getEtag(String cosmosDbPayload) {
     }
 
     public static <T> Page<T> parseDocuments(String cosmosDbPayload, Class<T> documentType) {
-        JsonObject objects = sParser.parse(cosmosDbPayload).getAsJsonObject();
-        JsonArray array = objects.get(Constants.DOCUMENTS_FILED_NAME).getAsJsonArray();
+        JsonArray array;
+        try {
+            JsonObject objects = sParser.parse(cosmosDbPayload).getAsJsonObject();
+            array = objects.get(Constants.DOCUMENTS_FILED_NAME).getAsJsonArray();
+        } catch (Exception e) {
```

Comment:
RuntimException

Please read the details for example at https://www.geeksforgeeks.org/checked-vs-unchecked-exceptions-in-java/

Reply:
Change made